### PR TITLE
[IMP] pos_sale, sale, sale_crm, sale_management, sale_stock, website_…

### PR DESCRIPTION
**Pull request by hch-odoo**
task - https://www.odoo.com/web?#id=2027274&action=327&model=project.task&view_type=form&menu_id=4720 
pad - https://pad.odoo.com/p/r.40420e3e8f12f7bb37836ef9675cdc27

### CODE REVIEW

File: addons/sale/views/sale_views.xml [hunk 1]
```diff
@@ -288,7 +288,11 @@
                         </group>
                         <group>
                             <field name="validity_date" attrs="{'invisible': [('state', 'in', ['sale', 'done'])]}"/>
-                            <field name="confirmation_date" attrs="{'invisible': [('state', 'in', ['draft', 'sent', 'cancel'])], 'required': [('state', 'in', ['sale', 'done'])]}"/>
+                            <div class="o_td_label"> 
+                                <label for="create_date" string="Quotation Date" attrs="{'invisible': [('state', 'in', ['sale', 'done', 'cancel'])]}"/>
```
Comment (Feyensv) [inline addons/sale/views/sale_views.xml]: Don't refer to create_date for the date_order label. The string of the label can change, but it is confusing if we show the information of a field for another field when hovering on it.

```xml
<div class="o_td_label">
    <label for="date_order" string="Quotation Date" attrs="{'invisible': [('state', 'in', ['sale', 'done', 'cancel'])]}"/>
    <label for="date_order" string="Order Date" attrs="{'invisible': [('state', 'in', ['draft', 'sent', 'cancel'])]}"/>
</div>
```

Comment (hch-odoo) [reply addons/sale/views/sale_views.xml]: changes are done.

**Comment by dbo-odoo** [comment]
@hch-odoo 
looks good to me

can you rebase before i merge? thx :)

**Comment by mba-odoo** [comment]
@dbo-odoo , Done

**Comment by dbo-odoo** [comment]
@robodoo r+

**Comment by beledouxdenis** [comment]
Looks like there is a crash when opening Sales > Orders > Sales Team since this revision

screenshot-596243-master-47a25d runbot14 odoo com-2019 08 21-14-51-48

It might be due to `_graph_date_column` returning `date_order`
https://github.com/odoo/odoo/blob/0cc30bf08382128a77455a51d831ef45ccdfdb48/addons/sale/models/sales_team.py#L101

and the fact the `_graph_get_model` returns `sale.report`,
https://github.com/odoo/odoo/blob/0cc30bf08382128a77455a51d831ef45ccdfdb48/addons/sale/models/sales_team.py#L96

and there is no `date_order` in `sale.report`. Instead, there is just a `date`:
https://github.com/odoo/odoo/blob/0cc30bf08382128a77455a51d831ef45ccdfdb48/addons/sale/report/sale_report.py#L78

**Comment by dbo-odoo** [comment]
should be fixed by #36048

**Pull request closed without merging**